cf-java-logging-support-servlet/src/main/java/com/sap/hcp/cf/logging/servlet/filter/RequestLoggingFilter.java

```diff
@@ -22,147 +28,235 @@
 import com.sap.hcp.cf.logging.servlet.dynlog.DynamicLogLevelProcessor;
 
 /**
- * A simple servlet filter that logs HTTP request processing info.
- *
+ * <p>
+ * A simple servlet filter that logs HTTP request processing info. It will read
+ * several HTTP Headers and store them in the SLF4J MDC, so that all log
+ * messages created during request handling will have those additional fields.
+ * It will also instrument the request to generate a request log containing
+ * metrics such as request and response sizes and response time. This
+ * instrumentation can be disabled by denying logs from {@link RequestLogger}
```

Comment:
how do I find out how denying logs works?

Reply:
This is framework specific and works differently in Logback and Log4j2. Therefore I cannot give a description in the comment. I will add it to the documentation though. Since the wiki is in a different repository, that change cannot be part of this PR.

Reply:
Okay then this should be fine